docs/Training-LSTM.md

```diff
@@ -0,0 +1,47 @@
+# Using Recurrent Neural Network in ML-Agents
+
+## What are memories for?
+Has it ever happened to you that you enter a room to get something and 
```

Comment:
Have you ever entered a room to get something and immediately forgot what you were looking for?